search-app/docs/site.md

```diff
@@ -14,12 +14,15 @@ See also the [API Documentation](search_api_docs.html).
 
```

Comment:
I noticed some typos on https://cmr.sit.earthdata.nasa.gov/search/site/docs 

1) Two misspellings - Doucmentation
2) Change sentence to "How to access the search API via an HTTP client"